Atomic/syscontainers.py

```diff
@@ -847,7 +846,6 @@ def extract_system_container(self, img, destination):
         repo = self._get_ostree_repo()
         if not repo:
             return False
-        self.args.remote = None
         return self._checkout_system_container(repo, img, img, 0, False, destination=destination, extract_only=True)
```

Comment:
the function `update_system_container` sets `self.args.remote` before calling `_checkout_system_container`, this information is read from the `info` file.  In this way, if the container was installed with another rootfs, then we use the same rootfs for updating the container.